Lib/turtledemo/clock.py

```diff
@@ -12,29 +11,29 @@
 from turtle import *
 from datetime import datetime
 
-def jump(distanz, winkel=0):
+def jump(distanz, angle=0):
     penup()
-    right(winkel)
+    right(angle)
     forward(distanz)
-    left(winkel)
+    left(angle)
     pendown()
 
-def hand(laenge, spitze):
-    fd(laenge*1.15)
+def hand(spitze, tip):
```

Comment:
I meant change 'laenge' to 'length' and 'spitze' to 'tip'.  (and 'distanz' to 'distance' and 'winkel' to 'angle').  Please revert and don't do anything with names until I verify that we can change parameter names in this file.